debezium-connector-db2/pom.xml

```diff
@@ -0,0 +1,274 @@
+        </dependency>
+
+        <!-- Testing -->
+        <dependency>
+            <groupId>io.debezium</groupId>
+            <artifactId>debezium-embedded</artifactId>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.debezium</groupId>
+            <artifactId>debezium-core</artifactId>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.debezium</groupId>
+            <artifactId>debezium-embedded</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-log4j12</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>log4j</groupId>
+            <artifactId>log4j</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.easytesting</groupId>
+            <artifactId>fest-assert</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.confluent</groupId>
+            <artifactId>kafka-connect-avro-converter</artifactId>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>io.fabric8</groupId>
+                <artifactId>docker-maven-plugin</artifactId>
+                <configuration>
+                    <watchInterval>500</watchInterval>
+                    <logDate>default</logDate>
+                    <verbose>true</verbose>
+                    <images>
+                        <image>
+                            <!-- A Docker image using the SQL Server -->
+                            <name>${docker.filter}</name>
+                            <run>
+                                <namingStrategy>none</namingStrategy>
+                                <env>
+                                    <ACCEPT_EULA>Y</ACCEPT_EULA>
+                                    <SA_PASSWORD>${sqlserver.password}</SA_PASSWORD>
+                                    <MSSQL_PID>Standard</MSSQL_PID>
+                                    <MSSQL_AGENT_ENABLED>true</MSSQL_AGENT_ENABLED>
+                                </env>
+                                <ports>
+                                    <port>${sqlserver.port}:1433</port>
+                                </ports>
+                                <log>
+                                    <prefix>sqlserver</prefix>
+                                    <enabled>true</enabled>
+                                    <color>yellow</color>
+                                </log>
```

Comment:
SQL Server?

Reply:
Updated to DB2 configuration in the latest push.